src/app/pages/services/components/service-tftp/service-tftp.component.ts

```diff
@@ -75,29 +98,22 @@ export class ServiceTFTPComponent {
               protected _injector: Injector, protected _appRef: ApplicationRef,
               protected userService: UserService) {}
 
-  resourceTransformIncomingRestData(data: any) {
```

Comment:
Please test that permissions are working as expected. Guessing this piece of code was a workaround for an issue that no longer exists.

Reply:
as discussed, it appears that we need to invert the values both on load and save with the websocket api